src/Core/ServerSettings.cpp

```diff
@@ -462,7 +466,11 @@ namespace DB
     DECLARE(Double, primary_index_cache_size_ratio, DEFAULT_PRIMARY_INDEX_CACHE_SIZE_RATIO, R"(The size of the protected queue (in case of SLRU policy) in the primary index cache relative to the cache's total size.)", 0) \
     DECLARE(Double, primary_index_cache_prewarm_ratio, 0.95, R"(The ratio of total size of mark cache to fill during prewarm.)", 0) \
     DECLARE(String, vector_similarity_index_cache_policy, DEFAULT_VECTOR_SIMILARITY_INDEX_CACHE_POLICY, "Vector similarity index cache policy name.", 0) \
-    DECLARE(UInt64, vector_similarity_index_cache_size, DEFAULT_VECTOR_SIMILARITY_INDEX_CACHE_MAX_SIZE, "Size of cache for vector similarityindex in bytes. Zero means disabled.", 0) \
+    DECLARE(UInt64, vector_similarity_index_cache_size, DEFAULT_VECTOR_SIMILARITY_INDEX_CACHE_MAX_SIZE, R"(Size of cache for vector similarity indexes. Zero means disabled.
+
+    :::note
```

Comment:
This kind of notes could be removed from the docs and we could use `changeable_without_restart` from `system.server_settings`.

Reply:
And, for this specific setting, it's not implemented in code so it provides wrong and outdated information in the table

@shankar-iyer Could you have a look and add these last settings to changeable_settings, please?

Reply:
@Algunenano I will do as part of https://github.com/ClickHouse/clickhouse-docs/issues/3250